### PR TITLE
Update to support php 8.1

### PR DESCRIPTION
**Pull request by iwiznia**
Issue https://github.com/Expensify/Expensify/issues/190022

### CODE REVIEW

**Review by flodnv — COMMENTED** [review]
👍 pending comments on your other PRs

**Review by flodnv — APPROVED** [review]

File: .php-cs-fixer.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<?php
+
+// Exclude file inside external libs
```
Comment (cead22) [inline .php-cs-fixer.php:3]:
```suggestion
// Exclude files inside external libs directories
```

File: .php-cs-fixer.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+$finder = PhpCsFixer\Finder::create()
+    ->exclude('vendor')
+    ->exclude('externalLib')
+    ->in(__DIR__)
+    ;
```
Comment (cead22) [inline .php-cs-fixer.php:8]:
```suggestion
    ->in(__DIR__);
```

File: .php-cs-fixer.php (hunk 1 above)
Comment (cead22) [reply .php-cs-fixer.php:3]: Feel free to fix this one as well https://github.com/Expensify/PHP-Libs/pull/476/files#diff-ef867390e21235c28e24595707bf636e1cc9cad29cf8ee4ac405c564a9facdf6R3

File: .php-cs-fixer.php (hunk 2 above)
Comment (cead22) [reply .php-cs-fixer.php:8]: Interesting that the Config is instantiated with new now, but the Finder still uses the factory pattern

**Review by cead22 — APPROVED** [review]

**Pull request merged**